diff -rupN gettext-0.20.2/gettext-runtime/intl/libgnuintl.in.h gettext-0.20.2-new/gettext-runtime/intl/libgnuintl.in.h
--- gettext-0.20.2/gettext-runtime/intl/libgnuintl.in.h	2020-04-14 03:32:21.000000000 +0200
+++ gettext-0.20.2-new/gettext-runtime/intl/libgnuintl.in.h	2020-07-28 14:51:42.012066985 +0200
@@ -410,6 +410,7 @@ extern int vasprintf (char **, const cha
 #endif
 
 #if @HAVE_WPRINTF@
+#ifndef __cplusplus
 
 #undef fwprintf
 #define fwprintf libintl_fwprintf
@@ -434,6 +435,8 @@ extern int vswprintf (wchar_t *, size_t,
 
 #endif
 
+#endif
+
 #endif
 
 
